documentation/modules/con-securing-kafka-authorization.adoc

```diff
@@ -1,25 +1,28 @@
 // Module included in the following assemblies:
 //
-// assembly-kafka-authentication-and-authorization.adoc
+// assembly-securing-access.adoc
 
-[id='ref-kafka-authorization-{context}']
-= Authorization
+[id='con-securing-kafka-authorization-{context}']
+= Kafka authorization
 
 You can configure authorization for Kafka brokers using the `authorization` property in the `Kafka.spec.kafka` resource.
 If the `authorization` property is missing, no authorization is enabled.
-When enabled, authorization is applied to all enabled xref:assembly-configuring-kafka-broker-listeners-{context}[listeners].
+When enabled, authorization is applied to all enabled listeners.
 The authorization method is defined in the `type` field.
 
-You can configure:
+Supported authorization options:
 
 * Simple authorization
 * xref:assembly-oauth-authorization_str[OAuth 2.0 authorization] (if you are using OAuth 2.0 token based authentication)
-* xref:type-KafkaAuthorizationOpa-reference[Open Policy Agent authorization]
+* xref:type-KafkaAuthorizationOpa-reference[Open Policy Agent (OPA) authorization]
+
+.Kafka cluster authorization options
+image::kafka-authorization-config-options.png[options for kafks authorization configuration]
 
 == Simple authorization
```

Comment:
It is a bit weird that we have here the docs for the simple authorization but link elsewhere for OAuth and OPA. Can we move the simple authorization to the `@DescriptionFile` as well or something?

Reply:
simple user config now moved to api/io.strimzi.api.kafka.model.KafkaAuthorizationSimple.adoc